src/main/kotlin/traffic_simulation/Main.kt

```diff
@@ -21,7 +21,7 @@ fun main(args: Array<String>) {
 
 
 fun testScenarioWithInternList() {
-    val road: RoadNetwork = RoadNetwork(capacity = 7)
+    val road: RoadNetwork = RoadNetwork(capacity = 70)
```

Comment:
Mir schwebt vor, dass die RoadCapacity wie wir sie bisher hatten - z.B. die 7 einen realen Wert abbildet. Einen Wert der für jedes Straßennetz irgendwie bestimmt werden kann. 

Nun würde man als Anwender erwarten, dass diese reale Wert auch eingegeben werden sollte bei RoadCapacity in der Main. Dort einen Wert anzugeben, der das 10-Fache des realen Wert ist, erfordert von dem Anwender eine eigenständige Umrechnung - wenn auch eine banale. Informieren müsste man ihn über die Notwendigkeit der Umrechnung.

Weniger Fehleranfällig wäre es bestimmt, wenn man weiterhin den "realen" Wert für RoadCapacity angibt - also z.B. die 7. Die Faktorisierung um 10 würde dann erst intern in Funktionen irgendwo erfolgen. Anbieten würde sich die Berechnung der prozentualen Kapazitätsbelegung in `randomDelayByCapacityAndDemand(demandX: Int, capacity: Int): Boolean {` . Dort könnte man recht weit oben einfügen: `capacity = capacity * 10`.